app/menu/main-menu.js

```diff
@@ -0,0 +1,76 @@
+const {Menu} = require('electron');
+const electron = require('electron');
+const app = electron.app;
+
+const template = [
+  {
+    label: 'Edit',
+    submenu: [
+      {
+        role: 'undo'
+      },
+      {
+        role: 'redo'
+      },
+      {
+        type: 'separator'
+      },
+      {
+        role: 'cut'
+      },
+      {
+        role: 'copy'
+      },
+      {
+        role: 'paste'
+      },
+      {
+        role: 'delete'
+      },
+      {
+        role: 'selectall'
+      }
+    ]
+  },
+  {
+    label: 'Help',
+    submenu: [
+      {
+        label: 'FAQ',
+        click () { require('electron').shell.openExternal('https://lbry.io/faq') }
```

Comment:
True. I wasn't giving this part a lot of thought because I knew it would be at least today before it was merged. Just trying to get the general framework of the menu up for review. We should probably decide today on the full set of menu items and what they should do.